README.md

```diff
@@ -48,7 +48,14 @@ Both solidity compilers and `certora-cli` dependencies are cached between runs.
 Example:
 
 ```yaml
-name: Certora Prover Workflow
+name: Certora Prover Submission Workflow
+description: |-
+  This workflow submits Certora Prover jobs on the specified configurations. Once all
```

Comment:
Just to clarify: `submits... jobs on the specified configurations`. What configurations are You referring to? `certoraRun` arguments (`*.conf` file)?

Reply:
Yes, I'm referring to the `configurations` parameter of the `Certora/Certora/certora-run-action`:

```yaml
      - uses: Certora/certora-run-action@v1
        with:
          # Add your configurations as lines, each line is separated.
          # Specify additional options for each configuration by adding them after the configuration.
          configurations: |-
            tests/conf-verified.conf
            tests/conf-verified.conf --rule monotone --method "counter()"
            tests/conf-verified.conf --rule invertible
            tests/conf-verified.conf --method "counter()"
```